### ISSUE TITLE
Error: Unexpected "a" at position 108 in state NULL1

### ISSUE THREAD

**Issue by DDynamic** [Pterodactyl/Daemon#10]
```
00:20:50.939Z FATAL pterodactyl.daemon: (path=/server/power, method=PUT)
    Error: Unexpected "a" at position 108 in state NULL1
        at Parser.proto.charError (/home/ddynamic/Workspaces/wings/daemon/node_modules/jsonparse/jsonparse.js:84:16)
        at Parser.proto.write (/home/ddynamic/Workspaces/wings/daemon/node_modules/jsonparse/jsonparse.js:294:19)
        at Stream.<anonymous> (/home/ddynamic/Workspaces/wings/daemon/node_modules/JSONStream/index.js:21:12)
        at Stream.stream.write (/home/ddynamic/Workspaces/wings/daemon/node_modules/through/index.js:26:11)
        at write (/home/ddynamic/Workspaces/wings/daemon/node_modules/docker-modem/node_modules/readable-stream/lib/_stream_readable.js:623:24)
        at flow (/home/ddynamic/Workspaces/wings/daemon/node_modules/docker-modem/node_modules/readable-stream/lib/_stream_readable.js:632:7)
        at HttpDuplex.pipeOnReadable (/home/ddynamic/Workspaces/wings/daemon/node_modules/docker-modem/node_modules/readable-stream/lib/_stream_readable.js:664:5)
        at emitNone (events.js:80:13)
        at HttpDuplex.emit (events.js:179:7)
        at emitReadable_ (/home/ddynamic/Workspaces/wings/daemon/node_modules/docker-modem/node_modules/readable-stream/lib/_stream_readable.js:448:10)
00:20:50.943Z FATAL pterodactyl.daemon: (path=/server/power, method=PUT)
    Error: Unexpected "r" at position 109 in state NULL1
        at Parser.proto.charError (/home/ddynamic/Workspaces/wings/daemon/node_modules/jsonparse/jsonparse.js:84:16)
        at Parser.proto.write (/home/ddynamic/Workspaces/wings/daemon/node_modules/jsonparse/jsonparse.js:294:19)
        at Stream.<anonymous> (/home/ddynamic/Workspaces/wings/daemon/node_modules/JSONStream/index.js:21:12)
        at Stream.stream.write (/home/ddynamic/Workspaces/wings/daemon/node_modules/through/index.js:26:11)
        at write (/home/ddynamic/Workspaces/wings/daemon/node_modules/docker-modem/node_modules/readable-stream/lib/_stream_readable.js:623:24)
        at flow (/home/ddynamic/Workspaces/wings/daemon/node_modules/docker-modem/node_modules/readable-stream/lib/_stream_readable.js:632:7)
        at HttpDuplex.pipeOnReadable (/home/ddynamic/Workspaces/wings/daemon/node_modules/docker-modem/node_modules/readable-stream/lib/_stream_readable.js:664:5)
        at emitNone (events.js:80:13)
        at HttpDuplex.emit (events.js:179:7)
        at emitReadable_ (/home/ddynamic/Workspaces/wings/daemon/node_modules/docker-modem/node_modules/readable-stream/lib/_stream_readable.js:448:10)
```

I get this when trying to power on a server as well as several other errors of the `unexpected position` sort.

**Comment by DaneEveritt** [comment]
I'm wondering if your server's configuration JSON is corrupted. Can you run it though a parser for me?

**Comment by DDynamic** [comment]
`config.json` in the daemon folder is intact. I have the container ID set as `sftp`. Is that correct?

**Comment by DaneEveritt** [comment]
What about the specific server that this error is being thrown from. Does this error occur right when you press start, or is it after a few seconds?

**Comment by DDynamic** [comment]
Right when I press start, the server rapidly starts and stops for a few seconds then this error occurs.

**Comment by DaneEveritt** [comment]
Can I get some system details, unsure as to what the issue could be. Can you provide more of the log?

**Comment by DaneEveritt** [comment]
Can confirm the presence of this issue. I'm assuming something got changed in one of the modules, will investigate.

**Comment by DaneEveritt** [comment]
I believe I have closed this in f41cdcf and a68f7f8. No idea what changed to cause that error to occur, but it was clearly due to the modem.
